Find Out - Announcement

smithWEBtek opened the pull request:
Provides the Announcement part of Find Out feature.
This PR solves #355 


Why is this change necessary?
To divide up the feature into sensible chunks of work.

How does it address the issue?
This code encompasses the Announcement extension implementing Tags and Taggings for Find Out filter.

What side effects does it have?
The styling of the find-out page filter results is temporary.

mzagaja commented:
Just wanted to note I am waiting for #347 to be merged into develop before reviewing this since it incorporates that pull request and the diff on this one is huge versus develop. I would also entertain updating this pull request to be a PR against #347 to present a smaller diff set.

smithWEBtek commented:
After merging #347 into develop, I rebased this branch on develop and incorporated your suggestions from Event branch, so now there are actually less diff files when comparing Announcement to develop (25 files) versus Announcement to #347 which is a diff of 30 files.  So I'm hoping we can proceed with comparing this branch to develop, which now includes Event.

smithWEBtek commented on app/assets/stylesheets/refinery/find_out.scss:
```diff
@@ -115,4 +115,20 @@
+  /* temporary styling ****************************/
+  .find-out__announcement__image {
+    width: 8rem;
+    border: 5px solid rgb(35, 49, 34);
```
yes this will be gone! thanks for review
